R/entities.R

```diff
@@ -45,13 +45,15 @@ Region <- R6Class("Region",
                   inherit = AbstractDataset,
                   public = list(covid_regional_data_identifier = NA,
                                 cases_subregion_source = "region_level_1",
+                                data_args = NULL,
                                 initialize = function(name,
                                                       covid_regional_data_identifier = NA,
                                                       case_modifier = NA,
                                                       generation_time = NA,
                                                       incubation_period = NA,
                                                       reporting_delay = NA,
                                                       cases_subregion_source = "region_level_1",
+                                                      data_args = NULL,
```

Comment:
you need to store this value (e.g. ln 66) as sadly R6 won't automatically map a param onto the class variable of the same name.

Reply:
now done